core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowBasePlugin.scala

```diff
@@ -168,4 +207,24 @@ object TooManyImagesBuiltError {
     """.stripMargin
 }
 
+final case class ImageName(registry: Option[String] = None,
```

Comment:
How is it different from sbtdocker defined `ImageName`?

Reply:
Maybe you don't have to redefine it @andreaTP

Reply:
done